apps/internal/oauth/ops/accesstokens/tokens.go

```diff
@@ -30,6 +30,7 @@ type IDToken struct {
 	TenantID          string `json:"tid,omitempty"`
 	Subject           string `json:"sub,omitempty"`
 	UPN               string `json:"upn,omitempty"`
+	IpAddress         string `json:"ipaddr,omitempty"`
```

Comment:
```suggestion
	IPAddress         string `json:"ipaddr,omitempty"`
```
(following the Google [style guide](https://google.github.io/styleguide/go/decisions#initialisms))